packages/ActionBar/src/hooks/useColumnsArrangement/useColumnsArrangement.js

```diff
@@ -1,9 +1,48 @@
 import React from "react";
 import produce from "immer";
 
-export default function useColumnsArrangement(defaultOrder, disabledColumnIds = []) {
+const savedItem = {
+  VISIBILITY: "VISIBILITY",
+  ORDERS: "ORDERS",
+};
+
+function getLocalStorageKey(localStoragePrefix, typeOfSavedItem) {
```

Comment:
is this localStoragePrefix intended to solve issues like two people with different accounts using the same computer?

`useColumnArragment({ localStragePrefix: ${project}-${user.id}-fature-name})`?

would it worth describing in the example a best practice when creating these names? can be the case that in two different features within the same app rendering two tables, one overrides the other one localstoraage content because incorrect naming conventions? (this might be an edge case but hard bug to catch)

Reply:
Sure, I can add some comments, the consuming app should be responsible for making sure the keys are unique I think